hack/run-k8s-e2e.sh

```diff
@@ -135,9 +143,16 @@ if ./kubevirtci kubectl get storageprofile local; then
 fi
 }
 
+function make_control_plane_schedulable() {
```

Comment:
it seems like this belongs in the kubevirtci script during the `create_cluster` function.

Reply:
This is not needed for the other e2e tests. I didn't want to change the default k8s setting where the changes are not on use.